staging/src/k8s.io/component-base/logs/validate.go

```diff
@@ -22,20 +22,23 @@ import (
 	"strings"
 
 	"github.com/spf13/pflag"
+
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"k8s.io/component-base/config"
 )
 
-func ValidateLoggingConfiguration(o *Options) []error {
-	errs := []error{}
-	if o.LogFormat != DefaultLogFormat {
+func ValidateLoggingConfiguration(c *config.LoggingConfiguration, fldPath *field.Path) field.ErrorList {
+	errs := field.ErrorList{}
+	if c.Format != DefaultLogFormat {
 		allFlags := UnsupportedLoggingFlags(hyphensToUnderscores)
 		for _, fname := range allFlags {
 			if flagIsSet(fname, hyphensToUnderscores) {
-				errs = append(errs, fmt.Errorf("non-default logging format doesn't honor flag: %s", fname))
+				errs = append(errs, field.Invalid(fldPath.Child("Format"), c.Format, fmt.Sprintf("Non-default format doesn't honor flag: %s", fname)))
```

Comment:
use user-facing field names, e.g. `format`

Reply:
still outstanding, use `format`, not `Format`

Reply:
Oh, sorry I though I addressed it. Must have made a mistake when rebasing.